src/sgame/sg_bot_nav.cpp

```diff
@@ -323,13 +323,12 @@ void BotStandStill( gentity_t *self )
 
 bool BotJump( gentity_t *self )
 {
-	int staminaJumpCost;
-
 	if ( self->client->pers.team == TEAM_HUMANS )
 	{
-		staminaJumpCost = BG_Class( self->client->ps.stats[ STAT_CLASS ] )->staminaJumpCost;
+		int const* stats = self->client->ps.stats;
```

Comment:
I would use either:

```suggestion
		auto &stats = self->client->ps.stats;
```
or
```suggestion
		int (&stats)[ MAX_STATS ] = self->client->ps.stats;
```